megaparse/sdk/megaparse_sdk/endpoints/file_upload.py

```diff
@@ -1,7 +1,9 @@
 from typing import Optional
 
 from httpx import Response
+
 from megaparse.sdk.megaparse_sdk.client import MegaParseClient
+from megaparse.sdk.megaparse_sdk.config import UploadFileInput
```

Comment:
THis is is more of a config to the parser than file input

Reply:
Changed the name :)